microraiden/microraiden/webui/index.html

```diff
@@ -2,57 +2,404 @@
+ /*  padding-right: 40px;*/
+
+ }
+
+ .alert {
+   margin-left: 10px;
+   margin-right: 10px;
+ }
+
+ .channel {
+   padding-top: 15px;
+   padding-bottom: 90px;
+ }
+
+ .status-label {
+   font-weight: bold;
+   text-align: left;
+ }
+
+ .glyphicon.on-state-opened {
+   color: #77b300;
+   font-size: 60px;
+   padding-left: 90px;
+ }
+
+ .glyphicon.on-state-closed {
+   color: #cc0000;
+   font-size: 60px;
+   padding-left: 90px;
+ }
+
+ .glyphicon.on-state-settled,
+ .glyphicon.on-state-error {
+   color: #ff8800;
+   font-size: 50px;
+   padding-left: 90px;
+ }
+
+ .status {
+   text-align: left;
+   font-size: 20px;
+ }
+
+ .balance {
+   font-size: 20px;
+   text-align: left;
+   color: white;
+ }
+
+ .balance-icon {
+   font-size: 20px;
+   font-weight: bold;
+   text-align: left !important;
+   color: white;
+   padding-left: 0;
+ }
+
+ .balance-label {
+   font-weight: bold;
+   text-align: left;
+ }
+
+ .btn-default {
+   height: 40px;
+ }
+
+ .btn-bar {
+   padding-top: 15px;
+ }
+
+ .box-container .btn {
+   font-size: 16px !important;
+ }
+
+ .box-container .input-group {
+   font-size: 16px !important;
+ }
+
+ .box-container h3, .h3 {
+     font-size: 34px !important;
+     color: #000000;
+ }
+
+ .box-container h4, .h4 {
+     font-size: 20px !important;
+     color: #000000;
+ }
+
+ .box-container h5, .h5 {
+     font-size: 16px !important;
+     color: #000000;
+ }
+
+ h1, h2, h3, h4, h5, h6, .h1, .h2, .h3, .h4, .h5, .h6 {
+     font-family: "Roboto","Helvetica Neue",Helvetica,Arial,sans-serif !important;
+     font-weight: 500 !important;
+     line-height: 1.1 !important;
+ }
+
+ .text-danger {
+   color: #cc0000 !important;
+ }
+
+ .text-warning {
+   color: #ff8800 !important;
+ }
+
+ .text-primary {
+   color: #2a9fd6 !important;
+ }
+
+ .green-btn {
+   background-color: #77b300 !important;
+ }
+
+ .form-control[readonly] {
+   background-color: #ffffff;
+ }
+
+ div.tooltip-inner {
+ 	width: 150px;
+ }
+
+
+</style>
+
+
 </head>
 <body>
   <div class="box-container">
     <div class="panel panel-default">
       <div class="panel-heading">
         <img src="/js/raiden.png" class="logo" />
-        <h3>Transfer Requested</h3>
-      </div>
-      <div class="panel-body">
-        <h4>Read this article and support the
-          <span id="publisher" >publisher</span> -
-        </h4>
-        <h4>by transfering <strong id="amount"></strong> <strong class="tkn-symbol">TKN</strong> tokens.</h4>
-        <div class="account-dropdown form-horizontal">
-          <div class="row">
-        <div class="col-xs-2"></div>
-            <div class="col-xs-9">
-        <div class="input-group input-group">
-          <span class="input-group-addon account-label" data-toggle="tooltip" data-placement="left" title="This is your current web3 account">Account</span>
-          <select class="form-control account-select" id="accounts"></select>
+        <h3>Transfer of <strong id="amount"></strong> TKN requested</h3>
```

Comment:
Don't use TKN literal, surround it by `.tkn-symbol`, so it get updated by js if the symbol is different.